man/systemd.net-naming-scheme.xml

```diff
@@ -531,6 +531,22 @@
     particular version of systemd).</para>
   </refsect1>
 
+  <refsect1>
+    <title>Limit reading specific sysfs attributes</title>
+
+    <para>When creating names for network cards, some naming schemes use data from sysfs populated
+    by the kernel. This means that although a specific naming scheme in udev is picked,
+    the network card's name can still change when a new kernel version adds a new sysfs attribute.
+    For example if kernel starts setting <constant>phys_port_name</constant>, udev will append the
+    "<constant>n</constant><replaceable>phys_port_name</replaceable>" suffix to the device name.
+    Users can set an allowlist or denylist for these. To create an allowlist, the user needs to set
+    <varname>ID_NET_NAME_ALLOW=0</varname> for the device and then list the allowed attributes in the format
+    of <varname>ID_NET_NAME_ALLOW_<replaceable>SYSFSATTR</replaceable>=1</varname>. In case of a deny list,
+    the user needs to set <varname>ID_NET_NAME_ALLOW=1</varname> for the device and then list the denied
+    attributes in the format of <varname>ID_NET_NAME_ALLOW_<replaceable>SYSFSATTR</replaceable>=0</varname>.
+    By default, all attributes are allowed.</para>
```

Comment:
maybe add another paragraph with something like this:

```
Or in other words, ID_NET_NAME_ALLOW= defines the fallback policy on sysattr use, and takes a boolean. ID_NET_NAME_ALLOW_<attr>= defines the policy for a specific sysattr, also taking a boolean.
```

or maybe even put that first somehow, i.e. that you start out with the logic that is implemented, and follow up with how to use it. Right now ou only describe the latter.

Reply:
Hmm, if `ID_NET_NAME_ALLOW=1` is the default, why is the user required to set it in case of a deny list?

Also, please use consistent spelling: "denylist" or "deny list".